lib/tapioca/internal.rb

```diff
@@ -29,14 +28,16 @@
 require "tapioca/helpers/gem_helper"
 
 require "tapioca/helpers/git_attributes"
```

Comment:
I guess we can also move these helpers below the `runtime` and `sorbet_ext` requires, right?